lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -6447,29 +6424,31 @@ SILValue ADContext::promoteToDifferentiableFunction(
             thunk->isBare(), thunk->isTransparent(), thunk->isSerialized(),
             thunk->isDynamicallyReplaceable(), ProfileCounter(),
             thunk->isThunk());
+        // If new thunk does not exist: clone the old thunk body, wrap the
```

Comment:
A little awkward. How about:
```suggestion
        // If the thunk is newly created, clone the old thunk body, wrap the
```